Examples/Physics_applications/laser_ion/CMakeLists.txt

```diff
@@ -20,3 +30,14 @@ add_warpx_test(
     diags/diagInst/  # output
     OFF  # dependency
 )
+
+# We do not have an intervals parser for PICMI yet which would accept more than a single integer for the output period
```

Comment:
We should think about implementing an intervals parser for our PICMI diagnostics which can handle more than a single integer. It should be able to do the same as our app input: https://warpx.readthedocs.io/en/latest/usage/parameters.html#intervals-parser